tests/e2e/core-tests/specs/shopper/front-end-variable-product-updates.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+const {
+	shopper,
+	merchant,
+	createVariableProduct,
+} = require( '@woocommerce/e2e-utils' );
+
+let variablePostIdValue;
+
+const cartDialogMessage = 'Please select some product options before adding this product to your cart.';
+
+const runVariableProductUpdateTest = () => {
+	describe('Update variable product',() => {
```

Comment:
Perhaps the name needs to make clearer what the test is doing in case it is confused for a merchant updating the product details. Although in general all our shopper flows probably need more consistency on how we name them so I don't think this is a blocker for the PR.

Reply:
Agreed, it was a minor change, so I went ahead and made the change so it is clearer also in the test runs.